src/lsp/cobol_indent/indenter.ml

```diff
@@ -62,30 +62,22 @@ let indenter ~source_format (str:string) (rdl:indent_record list) range =
   String.concat "\n" strl
 
 (*indent a range of file, with the default indent_config*)
-let indent_range' ~dialect ~source_format ~range ~file =
-  let file_content = Ez_file.V1.EzFile.read_file file in
-  (*
-    Not satisfied with the `Cobol_preproc.fold_text_lines`,
-    this function has an argument which is the name of file,
-    so when using lsp, every time using the formatting,
-    we must save the file before, it is not convenient.
-     (* NB: not anymore. *)
-  *)
+let indent_range' ~dialect ~source_format ~range ~filename ~contents =
```

Comment:
I'm not sure why we had those primes here. Could you take the opportunity of this small PR to get rid of them?

Reply:
This is because `indent_range` is exposed in `Cobol_indent` as the same function, but prints the result for CLI usage. I can make the printing happen in `command_indent_range.ml` instead of `cobol_indent.ml` but I'd rather do it in a separate PR.

Reply:
Ok, let's delay that then.